README.md

```diff
@@ -1,19 +1,25 @@
 # triton-service-groups
 
+Triton Service Groups provide monitoring for Triton applications, automatically scaling to meet capacity needs and maintain performance at the lowest possible cost. Manage applications using the TSG API or with Terraform to scale resources on Triton including VMs and containers. 
```

Comment:
> resources on Triton including VMs and containers

It only works for HVMs and infrastructure containers. There are no others resources it can manage.